### ISSUE TITLE
[Feature Request] more project related actions

### ISSUE THREAD

**Issue by cezaryece** [muse-sequencer/muse#1039]
I am often working with large projects with many sources located in many locations.
After finishing my job I would like to archive my project with all used sources in some directory for future and I want to be sure I can open it without problems after few years.
So... it will very nice to have some tool to do this, like menu action named "Archive Project" or "Collect All Sources" or "Copy Project with Sources".

**Comment by oldcastlehq** [comment]
Indeed. Exchanging projects with someone is a painful process, specially if you work with a lot of samples.
